src/styles/main.scss

```diff
@@ -0,0 +1,14 @@
+@import "./utils.scss/variables";
+@import "./block.scss/clock";
+@import "./utils.scss/funktions.scss";
+
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
Don't reset margins and paddings using * selector. Better specify the exact tags you want to reset